CHANGELOG.md

```diff
@@ -6,9 +6,13 @@ All notable changes to Corral will be documented in this file. This project adhe
 
 ### Fixed
 
-- Fix  should work if there is no corral.json ([PR #57](https://github.com/ponylang/corral/pull/57))
-- Fix Corral run only looking for binaries on $PATH ([PR #56](https://github.com/ponylang/corral/pull/56))
-- Improved testing and more ([PR #60](https://github.com/ponylang/corral/pull/60))
+- Fix `corral run` should work if there is no corral.json ([PR #57](https://github.com/ponylang/corral/pull/57))
+- Fix `corral run` only looking for binaries on $PATH ([PR #56](https://github.com/ponylang/corral/pull/56))
+- ([PR #60](https://github.com/ponylang/corral/pull/60))
+  - #10: Added integration tests
```

Comment:
This isn't user-facing. It shouldn't be in the CHANGELOG,

Reply:
Ah, ok. I'll remove this one too. I decided the other fixes were not user-facing either.